server.ts

```diff
@@ -1,12 +1,27 @@
 // server.ts - Imports the app and sets it to listen on the indicated port.
 
+// Import the app from src
 import app from "./src/app";
 
+// Import morgan library for logging http requests
+import morgan from "morgan";
+
+// Read port from environment variables
 const PORT = process.env.PORT;
 
+// Read debugging mode from environment variables
+const DEBUG = process.env.DEBUG === "true" || false;
+
+// Set logging level based on debug mode and add morgan middleware to app
+const logging_level = DEBUG ? "dev" : "tiny";
+app.use(morgan(logging_level));
+
+// Have the app listen on the desired port
 const server = app.listen(PORT, () => {
   const HOST = process.env.HOSTNAME || "localhost";
   console.log(`Listening on ${HOST}:${PORT}`);
+  DEBUG ??
+    console.log("Running in debug mode. Turn of debugging for production.");
```

Comment:
Typo: off

Reply:
Also, perhaps we should just explain here how to "turn off" debug mode. Something like `Change the DEBUG environment variable to false (or remove it)` Thoughts? @GreeneGunnar14

Reply:
I added the explanation to the startup message.